src/generator/www/js/main.js

```diff
@@ -1,54 +1,32 @@
-$(document).ready(function() {
-
-  let $email =$("#email");
-  let $name  =$("#name"); 
-  let $url  =$("#url");
-  let $theme =$("#theme");
-
-  $("#btn").click(function() { 
-
-  var form = document.querySelector("form");
-  var fd = new FormData(form);
-
-  var files ={
-    "sessionfile":  $( "#sessionfile" )[0].files[0],
-    "speakerfile":$( "#speakerfile" )[0].files[0],
-    "trackfile":$( "#trackfile" )[0].files[0] ,
-    "locationfile":$( "#locationfile" )[0].files[0]
-  }
-  
-  for (var key in files) {
-    //console.log(key, files[key]);
-    fd.append(key, files[key]);
-  }
-
-  var obj={
-    name:$name.val(),
-    email:$email.val(),
-    url :$url.val(),
-    theme:$theme.val(),
-    files:fd
-    
-  };
-
-  $.ajax({
-    type: "POST",
-    url:  " ",
-    data: obj,
-    processData: false,
-    contentType: false,
-    
-    success :function(data){
-      console.log(obj);
-    }
-       
+$(document).ready( function() {
+
+  $('#btn').click( function() {
+    var form = new FormData($('form').get(0));
+
+    // form.append('email', $('#email').val());
+    // form.append('name', $('#name').val());
+    // form.append('url', $('#url').val());
+    // form.append('theme', $('#theme').val());
+    // form.append('speakerfile', {'0': {}});
+    // form.append('sessionfile', {'0': {}});
+    // form.append('eventfile', {'0': {}});
+    // form.append('sponsorfile', {'0': {}});
+    // form.append('trackfile', {'0': {}});
+    // form.append('locationfile', {'0': {}});
+
+    $.ajax({
+      url : '/generate',
+      type: 'POST',
+      method: 'POST',
+      data: form,
+      mimeType: 'multipart/form-data',
+      success: function(data, textStatus, jqXHR) {
+        console.log(data.status);
+      },
+      error: function (jqXHR, textStatus, errorThrown) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2758166233/issues/source?bid=3233857&fileBranchId=3359574#l26)